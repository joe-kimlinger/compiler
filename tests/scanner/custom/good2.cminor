// booleans
b: boolean	 = int;
true; false, True;
"0"
