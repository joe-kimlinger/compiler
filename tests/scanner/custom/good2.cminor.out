COMMENT
IDENTIFIER b
COLON
BOOLEAN
ASSIGN
IDENTIFIER int
SEMICOLON
TRUE
SEMICOLON
FALSE
COMMA
IDENTIFIER True
SEMICOLON
STRING_LITERAL 0
